src/Database/Schema/Column.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace TCG\Voyager\Database\Schema;
+
+use Doctrine\DBAL\Schema\Column as DoctrineColumn;
+use Doctrine\DBAL\Types\Type;
+
+abstract class Column
+{
+    protected static $availableTypes;
+    protected static $doctrineTypeCategories = [];
+    protected static $platformTypes = [];
+
+    public static function make(array $column)
+    {
+        $name = Identifier::validate($column['name'], 'Column');
+        $type = $column['type'];
+        $type = ($type instanceof Type) ? $type : Type::getType(trim($type));
+
+        $options = array_diff_key($column, ['name' => $name, 'type' => $type]);
+
+        return new DoctrineColumn($name, $type, $options);
+    }
+
+    /**
+     * @return array
+     */
+    public static function toArray(DoctrineColumn $column)
+    {
+        $columnArray = $column->toArray();
+        $columnArray['type'] = $columnArray['type']->getName();
+        $columnArray['oldName'] = $columnArray['name'];
+
+        return $columnArray;
+    }
+
+    /**
+     * @return array
+     */
+    public static function getTypes()
+    {
+        // Add custom names to be used for ambiguous types (simple_array etc...)
+        //   maybe do this in the view?
```

Comment:
Or this could be added as static property that could be added directly from a service provider?